app/models/like.rb

```diff
@@ -1,6 +1,10 @@
 class Like < ApplicationRecord
```

Comment:
Why doesn't this model `belongs_to` `Post`?

Reply:
Since this is a polimorphic relation you dont need to place post or comment since we are ussing just the object itself, as you can see on the rails guide here

https://guides.rubyonrails.org/association_basics.html?fbclid=IwAR2L57P3hjUktKtqizRK8vmsd3d7T8rOK__pZFQEFpA7tTKm4qvIQLQPKxI#polymorphic-associations